rally_ovs/plugins/ovs/scenarios/ovn_nb.py

```diff
@@ -62,26 +62,37 @@ def create_or_update_address_set(self, name, ipaddr, create = True):
         else:
             self._address_set_add_addrs(name, ipaddr)
 
+    @atomic.action_timer("ovn.create_or_update_port_group")
+    def create_or_update_port_group(self, name, port_list, create = True):
+        if (create):
+            self._port_group_add(name, port_list, atomic_action = False)
+        else:
+            self._port_group_set(name, port_list, atomic_action = False)
+
     @atomic.action_timer("ovn.create_port_acls")
-    def create_port_acls(self, lswitch, lports, addr_set):
+    def create_port_acls(self, acl_dev, lports, addr_set,
+                         acl_type="switch"):
         """
         create two acl for each logical port
         prio 1000: allow inter project traffic
         prio 900: deny all
         """
         match = "%(direction)s == \"%(lport)s\" && ip4.dst == %(address_set)s"
```

Comment:
It should be this way 

match = "%(direction)s == %(lport)s && ip4.dst == $%(address_set)s"

Please see this PR which I submitted to fix this issue in dceara/ovn-scale-testing repo

https://github.com/dceara/ovn-scale-test/pull/2

Reply:
@numansiddique Thx, updated the PR